proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -53,7 +53,6 @@ message GenericSignedBeaconBlock {
         SignedBlindedBeaconBlockDeneb blinded_deneb  = 8;
     }
     bool is_blinded = 100;
-    uint64 payload_value = 101 [deprecated = true];
```

Comment:
When removing a field, we should mark the ID as reserved so it is never used again.

Add `reserved 101; // Deprecated fields` somewhere to this message please